test/tests/ghost-basics/run.sh

```diff
@@ -27,4 +27,10 @@ _request() {
 . "$dir/../../retry.sh" '_request GET / --output /dev/null'
 
 # Check that /ghost/ redirects to setup (the image is unconfigured by default)
-_request GET '/ghost/' -I | grep -q '^Location: .*setup'
+if [[ $serverImage == ghost:1* ]]
+then
+	_request GET '/ghost/api/v0.1/authentication/setup/' | grep -q 'status":false'
```

Comment:
Yeah, it does seem that this API endpoint will be unlikely to change; we'll just deal with it when it comes.